htdocs/societe/card.php

```diff
@@ -1500,7 +1500,10 @@ function formatCustomerSelection (selection) {
 		print '<tr><td class="titlefieldcreate">'.$form->editfieldkey('ProspectCustomer', 'customerprospect', '', $object, 0, 'string', '', 1).'</td>';
 		print '<td class="maxwidthonsmartphone">';
 		$selected = (GETPOSTISSET('client') ?GETPOST('client', 'int') : $object->client);
-		print $formcompany->selectProspectCustomerType($selected);
+		$reshook = $hookmanager->executeHooks('prospectcustomerfield');
```

Comment:
Method in execute hook must be a method in camel expression.  Replace  
prospectcustomerfield 
into
selectProspectCustomerType